src/components/Badge/Badge.scss

```diff
@@ -2,9 +2,8 @@
 
 $height: rem(20px);
 $small-height: rem(18px);
-$extra-small-height: rem(16px);
 $horizontal-padding: spacing(tight);
-$vertical-padding: rem(3px);
+$vertical-padding: rem(2px);
```

Comment:
Padding creates a visual buffer in cases where the badge text wraps, but isn’t primarily responsible for the height of the badge.